app/users/delete/template.hbs

```diff
@@ -5,6 +5,6 @@
     updateButtonText=updateButtonText }}
   <div class="alert alert-danger">
     <span class="glyphicon glyphicon-warning-sign"></span>
-    {{t 'messages.areYouSureDelete' user=model.name}}
+    {{t 'messages.delete' name=(concat (t 'models.user.names.singular') " " model.name)}}
```

Comment:
Don't use concat here because the order could be different in different languages.  For user I think it would be fine to have the message be:
Are you sure you wish to delete <username>
so you can drop the 
```hbs
(concat (t 'models.user.names.singular') " " 
```